docs/platforms/javascript/common/apis.mdx

```diff
@@ -0,0 +1,941 @@
+---
+title: APIs
```

Comment:
Another random thing I spotted: The right sidebar is so large that it goes beyond the screen, however when the "current heading" updates while scrolling the highlighted heading doesn't scroll into view.